.ci/lint-docs.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```

Comment:
This script was added to make `markdownlint-cli` easier to call from `pre-commit`.